test/Combinatorics/Matroids/Matroids.jl

```diff
@@ -368,4 +368,13 @@
 	   @test_throws ArgumentError is_quotient(M2, M1)
 	   @test_throws ArgumentError is_quotient(Q2, M2)
    end
+    
+  @testset "matroid6 and matroid_hex" begin
+    M = fano_matroid() 
+    N = uniform_matroid(2, 4)
+    M1 = matroid_from_matroid6(matroid6(M))
+    N1 = matroid_from_matroid6(matroid6(N))
+    @test is_isomorphic(M, M1)
+    @test is_isomorphic(N, N1)
+  end
```

Comment:
This testblock claims to test both encodings, but only contains tests for `matroid6`. Could you add corresponding tests for `matroid_hex` as well?